### PR TITLE
VSM-566 Rearch smoke test for archiver stage copy

### PR DESCRIPTION
**Pull request by hcoverston**
- The archiver parameter -rearch_stage_copy identifies a copy
  number to be used for the stage if the file that is being
  rearchived is offline. This rearch.sh test makes sure this
  rearch is successful even if the copy identfied in the
  rearch_stage_copy is not a valid copy.

### CODE REVIEW

File: VSM-566/mcf [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# samfsconfig /dev/sd* > /etc/opt/vsm/mcf built this file.
```
Comment (nichenke) [inline VSM-566/mcf]: These hardcoded paths won't work on all nodes - we need to use the mcf already present from running vsm-config

File: VSM-566/vsm566.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import os
+from autotest.client import test, utils
+
+
+__author__ = '''support@versity.com (Versity, Inc.)'''
```
Comment (nichenke) [inline VSM-566/vsm566.py]: let's make these versity-dev@versity.com -- since support@ is for customers

File: VSM-566/vsm566.sh [hunk 3]
```diff
@@ -0,0 +1,440 @@
...
+END=0
+DURATION_MS=0
+
+FILESYS1='/mnt/qfs2'
+MILESYS1='Qfs2'
+FILESYS2='/mnt/qfs3'
+MILESYS2='Qfs3'
+
+# mkfs opts for each filesys
+SAMMKFS_OPTS[1]="-a 64"
```
Comment (nichenke) [inline VSM-566/vsm566.sh]: vsm-config already setups up 'samfs.cmd' that should prevent the need for these.

File: VSM-566/mcf (hunk 1 above)
Comment (hcoverston) [inline VSM-566/mcf]: I removed the file mcf in the mod because I did use the installed mcf.

**Comment by heemstra** [comment]
I would strongly suggest removing everything from vsm566.sh that is not strictly required for reproducing the problem described by VSM-566.  We don't need to replicate any tests that vsm_smoke is alraedy doing.

**Comment by nichenke** [comment]
This moved into VSM proper, closing.

**Pull request closed without merging**